aiohttp/web_urldispatcher.py

```diff
@@ -414,6 +414,65 @@ def __repr__(self) -> str:
         return f"<PlainResource {name} {self._path}>"
 
 
+class GroupPlainResource(Resource):
+    def __init__(self) -> None:
+        super().__init__()
+        self._resources: Dict[str, Resource] = {}
+
+    @property
+    def canonical(self) -> str:
+        return "/"
+
+    def freeze(self) -> None:
+        self._routes.clear()
+        for resource in self._resources.values():
+            self._routes.extend(cast(Iterator[ResourceRoute], resource))
+
+    def add_prefix(self, prefix: str) -> None:
```

Comment:
What happens when calling `add_prefix()` on the `PlainResource` in a group? How is the group going to updates its keys?

Reply:
The method will be called along the chain  router.add_prefix => grp.add_prefix => plain.add_prefix

Reply:
Problem is that `.add_resource()` appears to be part of the public API: https://docs.aiohttp.org/en/stable/web_reference.html#aiohttp.web.UrlDispatcher.add_resource

So, if a user calls it, then it returns the `PlainResource`. The `.add_prefx()` method is not listed in the documentation, so not sure if it is considered part of the public API or not: https://docs.aiohttp.org/en/stable/web_reference.html#aiohttp.web.PlainResource

Probably a question for @asvetlov.